carbonfive/raygun-rails
# Move Yarn and other required dependencies into a Brewfile for easier

Problem
=======

Yarn is missing as an external dependency.

Solution
========
* Add Yarn dependency and installation step using `brew`

Change summary:
---------------
Moved Yarn and existing required dependencies into a Brewfile.

Steps to Verify:
----------------
1. Uninstall `yarn`
1. Verify failure running `./bin/setup`
1. `brew bundle --no-lock`
1. Verify `./bin/setup` succeeds
